src/app/services/copiedForGitHubCommenting.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { map, flatMap, catchError } from 'rxjs/operators';
+import { throwError, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WorldChampionsService {
+
+  initialYear = 1950;
+  api = 'http://ergast.com/api/f1';
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  getWorldChampions(from: number, to: number) {
+
+    const offset = from - this.initialYear; // 55
+    const limit = to - from + 1; // 11
+
+    return this.http.get(`${this.api}/driverStandings/1.json?limit=${limit}&offset=${offset}`)
+    // return this.http.get('../assets/dummy-data/world-champions.json')
+    .pipe(
+      map( res => {
```

Comment:
`(res: <type>) => {}` if possible. I'd recommend creating an `interface` for it. Its a lot of overhead work for a small personal project, but for larger projects (and learning), it would be good for type safety which eventually makes your coding easier because your IDE tells you as you are making mistakes, rather than you having to troubleshoot later.